src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -330,12 +359,11 @@ public void updateWithExistingRecord(StudentAttributes originalStudent) {
             this.section = originalStudent.section;
         }
     }
-
-    @Override
-    public Student toEntity() {
-        return new Student(email, name, googleId, comments, course, team, section);
+    
+    public Object toEntity() {
```

Comment:
This prevents me from subclassing StudentAttributes in StudentsDbTest for a StudentAttributes that persists to Students, although I guess the decorator pattern can be used here